matching-core/src/main/java/com/graphhopper/matching/util/HmmProbabilities.java

```diff
@@ -67,33 +55,12 @@ public double emissionLogProbability(double distance) {
      *                       consecutive map matching candidates.
      * @param linearDistance Linear distance [m] between two consecutive GPS
      *                       measurements.
-     * @param timeDiff       time difference [s] between two consecutive GPS
-     *                       measurements.
      */
-    public double transitionLogProbability(double routeLength, double linearDistance,
-                                           double timeDiff) {
-        if (timeDiff == 0) {
```

Comment:
> So more generally I think we should even use path.getWeight()

Good idea. I think we're all agreed that it'd be good to allow the user to configure their transition metric preference (or at least base it on the algoOptions). Maybe a separate PR?